src/components/Navigation.res

```diff
@@ -16,7 +16,7 @@ let linkOrActiveApiSubroute = (~route) => {
   }
 }
 
-let githubHref = "https://github.com/reason-association/rescript-lang.org#rescript-langorg"
```

Comment:
Link to the page instead of the header in README

Reply:
I wonder what @ryyppy thinks about this one

Reply:
sure, as long as its clear on how to contribute to the docs.